apps/coscrad-frontend/src/components/resources/terms/term-detail.full-view.presenter.tsx

```diff
@@ -21,7 +21,8 @@ export const TermDetailFullViewPresenter = ({
             <div data-testid={id} />
             <SingleOptionalPropertyPresenter display="Contributor" value={contributor} />
             <div id="media-player">
-                <MediaPlayer listenMessage="Play!" audioUrl={audioURL} />
+                {/* <MediaPlayer listenMessage="Play!" audioUrl={audioURL} /> */}
+                <AudioClipPlayer audioUrl={audioURL} PlayPauseButton={DefaultPlayPauseButton} />
```

Comment:
We should rename the prop `PlayButton` and remove the need to provide the default.